infra/modules/aws/main.tf

```diff
@@ -70,7 +71,12 @@ resource "aws_iam_role" "api-caller" {
           "Effect": "Allow",
           "Action": "execute-api:Invoke",
           "Resource": "arn:aws:execute-api:*:${var.aws_account_id}:*/*/GET/*",
-        }
+        },
+        {
+          "Effect": "Allow",
+          "Action": "execute-api:Invoke",
+          "Resource": "arn:aws:execute-api:*:${var.aws_account_id}:*/*/HEAD/*",
```

Comment:
I don't understand the point in allowing `HEAD` ... seems to complicate things, without adding any practical value

Reply:
Well, HEAD is slightly faster than GET, but other than that analogous and harmless from the proxy POV.
I could have changed the HEAD calls in Worklytics, but we should adapt proxy to Worklytics, not the other way around.